game-core/src/main/java/games/strategy/triplea/ui/screen/Tile.java

```diff
@@ -28,12 +25,10 @@
 
 public class Tile {
   public static final LockUtil LOCK_UTIL = LockUtil.INSTANCE;
-  private static final boolean DRAW_DEBUG = false;
-  private static final Logger logger = Logger.getLogger(Tile.class.getName());
 
-  // allow the gc to implement memory management
-  private SoftReference<Image> imageRef;
```

Comment:
This is about the only change I'm not 100% sure on. Are we removing having a soft reference to the image for tiles on purpose? Or is this now handled somewhere else? If we are removing it then could this become a problem for really large maps with 1000s of tiles?